Dockerfile

```diff
@@ -55,7 +55,7 @@ RUN git clone https://github.com/NVIDIA/apex.git && \
 # install megatron core, this can be removed once 0.3 pip package is released
 RUN git clone https://github.com/NVIDIA/Megatron-LM.git && \
   cd Megatron-LM && \
-  git checkout 84f64880b3651c4f7cf90da337ee4e7d9968acab && \
+  git checkout 1b1a798bca149ecf98d43b6082cd1ee80079eadf && \
```

Comment:
Which mcore commit is this? Does it contain the dist ckpt fixes?

Reply:
this is the most recent one https://github.com/NVIDIA/Megatron-LM/commit/1b1a798bca149ecf98d43b6082cd1ee80079eadf

will need to update this once my dist ckpt PR is merged and synced on github.

Reply:
updated mcore with my dist ckpt fix.